guides/developer-guide.md

```diff
@@ -1,17 +1,9 @@
-# Developer startup guide
+# OctoBot architecture
 
-{% hint style="note" %}
+{% hint style="info" %}
 This page is referring to OctoBot in versions superior to 0.4.0.
```

Comment:
I think we can remove it, there is no 0.3.X running anymore, what do you think @GuillaumeDSM?